<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络速度监控</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>网络接口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="interfaceComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>IP地址</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>下载速度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="downloadSpeedLabel">
        <property name="text">
         <string>0 B/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>上传速度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uploadSpeedLabel">
        <property name="text">
         <string>0 B/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>总速度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalSpeedLabel">
        <property name="text">
         <string>0 B/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>总流量统计</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>总下载:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalDownloadLabel">
           <property name="text">
            <string>0 B</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>总上传:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalUploadLabel">
           <property name="text">
            <string>0 B</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>总流量:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalDataLabel">
           <property name="text">
            <string>0 B</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
